klog.go

```diff
@@ -436,6 +440,7 @@ func InitFlags(flagset *flag.FlagSet) {
 	flagset.BoolVar(&logging.oneOutput, "one_output", logging.oneOutput, "If true, only write logs to their native severity level (vs also writing to each lower severity level)")
 	flagset.BoolVar(&logging.skipLogHeaders, "skip_log_headers", logging.skipLogHeaders, "If true, avoid headers when opening log files")
 	flagset.Var(&logging.stderrThreshold, "stderrthreshold", "logs at or above this threshold go to stderr")
+	flagset.Var(&logging.alsoToStderrThreshold, "alsotostderrthreshold", "logs at or above this threshold go to stderr as well as files")
```

Comment:
i personally find the new flag confusing but i can see you are trying to fix an already confusing behavior with it.

Reply:
I see no other way to fix this without breaking the old installations.